src/main/java/io/jenkins/plugins/gitlabbranchsource/ProjectNamingStrategyTrait.java

```diff
@@ -18,8 +18,8 @@ public class ProjectNamingStrategyTrait extends SCMNavigatorTrait {
     private int strategyId = 1;
 
     @DataBoundConstructor
-    public ProjectNamingStrategyTrait() {
-        // empty
+    public ProjectNamingStrategyTrait(int strategyId) {
+        this.strategyId = strategyId;
```

Comment:
Please revert.

As this has a data bound setter and should not be inside the `DataBoundConstructor`